lib/yard/code_objects/constant_object.rb

```diff
@@ -12,5 +12,12 @@ class ConstantObject < Base
     def value=(value)
       @value = format_source(value)
     end
+
+    # The scope of the constants is always +:class+
+    #
+    # @return [Symbol] +:class+
+    def scope
```

Comment:
This probably happens because of some shared logic in the templates. You may want to change the code in the template itself rather than adding new APIs. I'm not sure a #scope method makes sense for a ConstantObject.